crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -692,6 +791,54 @@ where
     }
 }
 
+pub struct CloneBundle<T> {
+    pub source: Entity,
+    pub target: Entity,
+    _bundle: PhantomData<T>,
+}
+
+impl<T> CloneBundle<T> {
+    /// Creates a new [`CloneBundle`] with given source and target IDs.
+    pub fn new(source: Entity, target: Entity) -> Self {
+        Self {
+            source,
+            target,
+            _bundle: PhantomData,
+        }
+    }
+}
+
+impl<T> Command for CloneBundle<T>
+where
+    T: Bundle + Clone + 'static,
+{
+    fn write(self, world: &mut World) {
+        let mut source_mut = if let Some(some) = world.get_entity_mut(self.source) {
+            some
+        } else {
+            panic!("Could not clone a bundle (of type `{}`) from entity {:?} because it doesn't exist in this World.\n\
+            If this command was added to a newly spawned entity, ensure that you have not despawned that entity within the same stage.\n\
+            This may have occurred due to system order ambiguity, or if the spawning system has multiple command buffers", std::any::type_name::<T>(), self.source);
+        };
+
+        let bundle = if let Some(some) = source_mut.remove_bundle::<T>() {
+            some
+        } else {
+            return;
+        };
+        source_mut.insert_bundle(bundle.clone());
```

Comment:
This will trigger "added" and "changed" events for each component in the bundle, which is undesirable for a clone operation.